platforms/hyperledger-indy/configuration/samples/network-indyv3.yaml

```diff
@@ -11,80 +15,117 @@ network:
     retry_count: 20                 # Retry count for the checks
     external_dns: disabled           # Should be enabled if using external-dns for automatic route configuration
 
+  # Docker registry details where images are stored. This will be used to create k8s secrets
+  # Please ensure all required images are built and stored in this registry.
+  # Do not check-in docker_password.
   docker:
     url: "docker_url"
     username: "docker_username"
     password: "docker_password"
 
   name: baf
 
+  # Informatio about pool transaction genesis and domain transactions genesis
   genesis:
     state: absent
     pool: genesis/pool_transactions_genesis
     domain: domain/domain_transactions_genesis
 
+  # Allows specification of one or many organizations that will be connecting to a network.
   organizations:
+    # Specification for the 1st organization. Each organization maps to a VPC and a separate k8s cluster
     - organization:
       name: authority
       type: peer
       cloud_provider: aws-baremetal
```

Comment:
mention why cloud_provider is baremetal. ie aws eks is not supported for some reason.